mne/utils/tests/test_numerics.py

```diff
@@ -465,3 +465,19 @@ def test_julian_conversions():
 
         assert (jd == _dt_to_julian(dd))
         assert (jd == _cal_to_julian(cal[0], cal[1], cal[2]))
+
+
+def test_grand_average_empty_sequence():
+    with pytest.raises(ValueError, match='Please pass a list of Evoked'):
+        grand_average([])
+
+
+def test_grand_average_len_1():
+    base_dir = op.join(op.dirname(__file__), '..', '..', 'io', 'tests',
+                       'data')
+    fname = op.join(base_dir, 'test-ave.fif')
+    evokeds = read_evokeds(fname, condition=[0], proj=True)  # list of length 0
+
+    with pytest.warns(RuntimeWarning, match='Only a single evoked'):
+        gave = grand_average(evokeds)
+        assert_allclose(gave.data, evokeds[0].data)
```

Comment:
Usually you put only what is necessary inside context managers, it makes clear what needs them. So this line should be unindented